### PR TITLE
docs(03-router): minor adjustments

### PR DESCRIPTION
**Pull request by bighappyface**
This PR proposes a handful of quick adjustments for clarity.

### CODE REVIEW

File: 03-router.md [hunk 1]
```diff
@@ -137,13 +137,13 @@ You guessed it! Still inside `menu_get_item()`.
...
 
 To start, it needs to translate placeholders (i.e., "%" signs) in paths to the loaded entity if a load function exists for it. For example, if our path in `menu_router` is `node/%node` then this chunk of code will say "ok, I see `%node` as opposed to just `%`, therefore, I know I need to run `node_load()` on whatever is in that part of the URL, which in this case is `1234`. 
 
-To do this, it runs [`_menu_load_objects()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_load_objects/7) which calls the appropriate load function if there is one and, assuming we don't get an access error or a missing entity or anything else which would throw an error, put the result (i.e., the fully loaded entity) back into the menu router item for later use. 
+To do this, it runs [`_menu_load_objects()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_load_objects/7) which calls the appropriate load function if there is one and, assuming we don't get an access error, or a missing entity, or anything else which would throw an error, put the result (i.e., the fully loaded entity) back into the menu router item for later use.
```
Comment (rhuffstedtler) [inline 03-router.md]: If you're going to do a serial list, you need to omit the or immediately after access error. What Mike has is grammatically correct, but I think it reads better as a serial list.

File: 03-router.md [hunk 2]
```diff
@@ -137,13 +137,13 @@ You guessed it! Still inside `menu_get_item()`.
...
+To do this, it runs [`_menu_load_objects()`](https://api.drupal.org/api/drupal/includes%21menu.inc/function/_menu_load_objects/7) which calls the appropriate load function if there is one and, assuming we don't get an access error, or a missing entity, or anything else which would throw an error, put the result (i.e., the fully loaded entity) back into the menu router item for later use. 
 
-As a side note, this is a very common place to hit an access denied, in cases where the user doesn't have access to whichever entity we're trying to load. For example, an anonymous user trying to view an unpublished node would fail at this step.
+As a side note, this is a very common place to hit an _access denied_, in cases where the user doesn't have access to whichever entity we're trying to load. For example, an anonymous user trying to view an unpublished node would fail at this step.
```
Comment (rhuffstedtler) [inline 03-router.md]: Is access_denied an error or an exception? We should probably make that explicit.

**Comment by bighappyface** [comment]
@rhuffstedtler thanks for the review. I have updated the serial list as recommended, and referred to the _access denied_ as "_access denied_ page." I don't know that referring to it as an error or exception was the best way to describe it, and most people have see the "access denied" page that the text is referring to.

Thoughts?

**Comment by crittermike** [comment]
I think this looks great now. Huge thanks to both @bighappyface for the PR and @rhuffstedtler for the review. Merging, but feel free to reopen if you're not happy with it @rhuffstedtler.

**Comment by rhuffstedtler** [comment]
@bighappyface - yep, that makes sense.

**Pull request merged**